pkg/controller/credentialsrequest/status.go

```diff
@@ -276,11 +276,20 @@ func findClusterOperatorCondition(conditions []configv1.ClusterOperatorStatusCon
 // buildExpectedRelatedObjects returns the list of expected related objects, used
 // by the oc must-gather command to fetch resource yaml for debugging purposes.
 // Keeping this up to date across versions via the code seems like the safest option.
-func buildExpectedRelatedObjects() []configv1.ObjectReference {
-	return []configv1.ObjectReference{
+func buildExpectedRelatedObjects(credRequests []minterv1.CredentialsRequest) []configv1.ObjectReference {
+	related := []configv1.ObjectReference{
 		{
 			Resource: "namespaces",
 			Name:     cloudCredOperatorNamespace,
 		},
 	}
+	for _, cr := range credRequests {
+		related = append(related, configv1.ObjectReference{
+			Group:     minterv1.SchemeGroupVersion.Group,
+			Resource:  "CredentialsRequest",
```

Comment:
the resource is wrong, this should be `credentialsrequests`

Reply:
https://github.com/openshift/cloud-credential-operator/blob/77eaf88b08645f1953b657bf0109aab15054c50b/manifests/00_v1_crd.yaml#L12

Reply:
How is the `plural` field more correct to use here than the `kind` field?  Is there an API that using kind violates, or a tool that it breaks?  For what it's worth, I tested this change and verified that must-gather gathered the CredentialsRequest resources.  Morever, use the kind (upper camel case) in the ingress operator too have not seen or received reports of any problems.